apps/web/src/pages/DirectivePage/DirectivePage.tsx

```diff
@@ -0,0 +1,474 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import {
+  MapIcon,
+  ChartPieIcon,
+  NewspaperIcon,
+  MagnifyingGlassCircleIcon,
+  BookmarkSquareIcon,
+} from "@heroicons/react/24/outline";
+
+import {
+  Heading,
+  ExternalLink,
+  Accordion,
+  CardFlat,
+} from "@gc-digital-talent/ui";
+import { Locales, useLocale } from "@gc-digital-talent/i18n";
+
+import Hero from "~/components/Hero";
+import CallToActionLink from "~/components/CallToAction/CallToActionLink";
+import useBreadcrumbs from "~/hooks/useBreadcrumbs";
+import useRoutes from "~/hooks/useRoutes";
+
+import talentPlanEn from "~/assets/documents/Forward_Talent_Plan_EN.docx";
+import talentPlanFr from "~/assets/documents/Plan_prospectif_sur_les_talents_FR.docx";
+import recruitmentEn from "~/assets/documents/Digital_Recruitment_Template_EN.docx";
+import recruitmentFr from "~/assets/documents/Modele_de_recrutement_numerique_FR.docx";
+import guidanceEn from "~/assets/documents/Enabling_Conditions_Guidance_EN.docx";
+import guidanceFr from "~/assets/documents/Orientation_sur_les_conditions_habilitantes_FR.docx";
+import contractingEn from "~/assets/documents/Digital_Contracting_Questionnaire_EN.docx";
+import contractingFr from "~/assets/documents/Questionnaire_d'octroi_de_contrats_numeriques_FR.docx";
+
+import { getFormLinks, getGenericLinks } from "./utils";
+
+const policyLink = (locale: Locales, chunks: React.ReactNode) => (
+  <ExternalLink
+    newTab
+    href={
+      locale === "en"
+        ? "https://www.tbs-sct.canada.ca/pol/doc-eng.aspx?id=32603"
+        : "https://www.tbs-sct.canada.ca/pol/doc-fra.aspx?id=32603"
+    }
+  >
+    {chunks}
+  </ExternalLink>
+);
+
+const contactLink = (chunks: React.ReactNode) => (
+  <a href="mailto:GCTalentGC@tbs-sct.gc.ca">{chunks}</a>
+);
+
+const DirectivePage = () => {
+  const intl = useIntl();
+  const { locale } = useLocale();
+  const paths = useRoutes();
+
+  const pageTitle = intl.formatMessage({
+    defaultMessage: "Directive on Digital Talent",
+    id: "xXwUGs",
+    description: "Title for the digital talent directive page",
+  });
+
+  const crumbs = useBreadcrumbs([
+    {
+      label: pageTitle,
+      url: paths.directive(),
+    },
+  ]);
+
+  const directiveUrl =
+    locale === "en"
+      ? "https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32749"
+      : "https://www.tbs-sct.gc.ca/pol/doc-fra.aspx?id=32749";
+
+  const readDirectiveMessage = intl.formatMessage({
+    defaultMessage: "Read the Directive",
```

Comment:
LOL, I believe this is the copy you received but "the Directive" sounds so ominous and dystopian.  :laughing: